common/djangoapps/adaptive_learning/config/models.py

```diff
@@ -0,0 +1,21 @@
+"""
+Provides the configuration models for Adaptive Learning
+"""
+
+from config_models.models import ConfigurationModel
+
+
+class AdaptiveLearningEnabledFlag(ConfigurationModel):
+    """
+    A flag that enables/disables Adaptive Learning across an entire instance.
+    """
+
+    class Meta(object):
+        app_label = "adaptive_learning"
+
+    def __unicode__(self):
+        current_model = AdaptiveLearningEnabledFlag.current()
+        return u"AdaptiveLearningEnabledFlag: enabled {}".format(
+            current_model.is_enabled()
```

Comment:
I think you can just use `self.enabled` here, since `is_enabled` just does this:

```python
return cls.current(*key_fields).enabled
```